beacon-chain/core/light-client/lightclient.go

```diff
@@ -278,24 +278,40 @@ func createDefaultLightClientUpdate(currentSlot primitives.Slot) (interfaces.Lig
 	var m proto.Message
 	if currentEpoch < params.BeaconConfig().CapellaForkEpoch {
 		m = &pb.LightClientUpdateAltair{
+			AttestedHeader:          &pb.LightClientHeaderAltair{},
```

Comment:
Added an empty header here. because the `return light_client.NewWrappedHeader(m)` would throw an error otherwise.